lib/smart_answer_flows/marriage-abroad/outcomes/os_affirmation/_fees.govspeak.erb

```diff
@@ -4,7 +4,7 @@
 
 <% unless calculator.country_without_consular_facilities? %>
   <% if calculator.ceremony_country == 'vietnam' %>
-    <%= render partial: 'consular_fees_in_cash_visa_master.govspeak.erb',
+    <%= render partial: 'link_to_consular_fees_for_vietnam.govspeak.erb',
```

Comment:
I've removed the commit that renamed this partial. I considered renaming the partial so that it followed the rename in PR #2592 but figured that would probably make merging #2592 harder.